diff --git a/node_modules/node-fetch/lib/index.es.js b/node_modules/node-fetch/lib/index.es.js
index aae9799..70a6b29 100644
--- a/node_modules/node-fetch/lib/index.es.js
+++ b/node_modules/node-fetch/lib/index.es.js
@@ -3,7 +3,7 @@ process.emitWarning("The .es.js file is deprecated. Use .mjs instead.");
 import Stream from 'stream';
 import http from 'http';
 import Url from 'url';
-import whatwgUrl from 'whatwg-url';
+import { createRequire } from 'module';
 import https from 'https';
 import zlib from 'zlib';
 
@@ -14,6 +14,16 @@ const Readable = Stream.Readable;
 
 const BUFFER = Symbol('buffer');
 const TYPE = Symbol('type');
+const require = createRequire(import.meta.url);
+let whatwgUrl;
+
+function getWhatwgUrl() {
+	if (whatwgUrl === undefined) {
+		whatwgUrl = require('whatwg-url');
+	}
+
+	return whatwgUrl;
+}
 
 class Blob {
 	constructor() {
@@ -1138,7 +1148,7 @@ Object.defineProperty(Response.prototype, Symbol.toStringTag, {
 });
 
 const INTERNALS$2 = Symbol('Request internals');
-const URL = Url.URL || whatwgUrl.URL;
+const URL = Url.URL || getWhatwgUrl().URL;
 
 // fix an issue where "format", "parse" aren't a named export for node <10
 const parse_url = Url.parse;
@@ -1397,7 +1407,7 @@ AbortError.prototype = Object.create(Error.prototype);
 AbortError.prototype.constructor = AbortError;
 AbortError.prototype.name = 'AbortError';
 
-const URL$1 = Url.URL || whatwgUrl.URL;
+const URL$1 = Url.URL || getWhatwgUrl().URL;
 
 // fix an issue where "PassThrough", "resolve" aren't a named export for node <10
 const PassThrough$1 = Stream.PassThrough;
diff --git a/node_modules/node-fetch/lib/index.js b/node_modules/node-fetch/lib/index.js
index 567ff5d..61a0dea 100644
--- a/node_modules/node-fetch/lib/index.js
+++ b/node_modules/node-fetch/lib/index.js
@@ -7,7 +7,6 @@ function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'defau
 var Stream = _interopDefault(require('stream'));
 var http = _interopDefault(require('http'));
 var Url = _interopDefault(require('url'));
-var whatwgUrl = _interopDefault(require('whatwg-url'));
 var https = _interopDefault(require('https'));
 var zlib = _interopDefault(require('zlib'));
 
@@ -18,6 +17,15 @@ const Readable = Stream.Readable;
 
 const BUFFER = Symbol('buffer');
 const TYPE = Symbol('type');
+let whatwgUrl;
+
+function getWhatwgUrl() {
+	if (whatwgUrl === undefined) {
+		whatwgUrl = _interopDefault(require('whatwg-url'));
+	}
+
+	return whatwgUrl;
+}
 
 class Blob {
 	constructor() {
@@ -1142,7 +1150,7 @@ Object.defineProperty(Response.prototype, Symbol.toStringTag, {
 });
 
 const INTERNALS$2 = Symbol('Request internals');
-const URL = Url.URL || whatwgUrl.URL;
+const URL = Url.URL || getWhatwgUrl().URL;
 
 // fix an issue where "format", "parse" aren't a named export for node <10
 const parse_url = Url.parse;
@@ -1401,7 +1409,7 @@ AbortError.prototype = Object.create(Error.prototype);
 AbortError.prototype.constructor = AbortError;
 AbortError.prototype.name = 'AbortError';
 
-const URL$1 = Url.URL || whatwgUrl.URL;
+const URL$1 = Url.URL || getWhatwgUrl().URL;
 
 // fix an issue where "PassThrough", "resolve" aren't a named export for node <10
 const PassThrough$1 = Stream.PassThrough;
diff --git a/node_modules/node-fetch/lib/index.mjs b/node_modules/node-fetch/lib/index.mjs
index 2863dd9..4fafea7 100644
--- a/node_modules/node-fetch/lib/index.mjs
+++ b/node_modules/node-fetch/lib/index.mjs
@@ -1,7 +1,7 @@
 import Stream from 'stream';
 import http from 'http';
 import Url from 'url';
-import whatwgUrl from 'whatwg-url';
+import { createRequire } from 'module';
 import https from 'https';
 import zlib from 'zlib';
 
@@ -12,6 +12,16 @@ const Readable = Stream.Readable;
 
 const BUFFER = Symbol('buffer');
 const TYPE = Symbol('type');
+const require = createRequire(import.meta.url);
+let whatwgUrl;
+
+function getWhatwgUrl() {
+	if (whatwgUrl === undefined) {
+		whatwgUrl = require('whatwg-url');
+	}
+
+	return whatwgUrl;
+}
 
 class Blob {
 	constructor() {
@@ -1136,7 +1146,7 @@ Object.defineProperty(Response.prototype, Symbol.toStringTag, {
 });
 
 const INTERNALS$2 = Symbol('Request internals');
-const URL = Url.URL || whatwgUrl.URL;
+const URL = Url.URL || getWhatwgUrl().URL;
 
 // fix an issue where "format", "parse" aren't a named export for node <10
 const parse_url = Url.parse;
@@ -1395,7 +1405,7 @@ AbortError.prototype = Object.create(Error.prototype);
 AbortError.prototype.constructor = AbortError;
 AbortError.prototype.name = 'AbortError';
 
-const URL$1 = Url.URL || whatwgUrl.URL;
+const URL$1 = Url.URL || getWhatwgUrl().URL;
 
 // fix an issue where "PassThrough", "resolve" aren't a named export for node <10
 const PassThrough$1 = Stream.PassThrough;
